db/migrate/20170207200029_add_sti_type_to_configuration_script_source.rb

```diff
@@ -0,0 +1,5 @@
+class AddStiTypeToConfigurationScriptSource < ActiveRecord::Migration[5.0]
+  def change
+    add_column :configuration_script_sources, :type, :string
```

Comment:
We don't need migration because the coming release is the first one with this table.